.github/workflows/ci.yml

```diff
@@ -170,6 +170,8 @@ jobs:
             - { target: thumbv7em-none-eabi,              os: ubuntu-latest,                    std: 1 }
             - { target: thumbv7em-none-eabihf,            os: ubuntu-latest,                    std: 1 }
             - { target: thumbv7m-none-eabi,               os: ubuntu-latest,                    std: 1 }
+            
+            - { target: cross                             os: ubuntu-latest }
```

Comment:
This won't work either sadly, it'll have to be it's own task/action for the image to be built and published